snuba/utils/streams/kafka_consumer_with_commit_log.py

```diff
@@ -2,8 +2,8 @@
 
 from arroyo import Message, Partition, Topic
 from arroyo.backends.kafka import KafkaConsumer, KafkaPayload
-from arroyo.retries import RetryPolicy
 from arroyo.synchronized import Commit, commit_codec
+from arroyo.utils.retries import RetryPolicy
```

Comment:
This is a small thing to think about, but with the SDK I eventually collapsed the imports so you could import everything from the base package, e.g.

```
from snuba_sdk import Column, Function...
```
instead of 
```
from snuba_sdk.column import Column
from snuba_sdk.function import Function
```
I don't think arroyo should expose everything at the top level, but it might be worth allowing importing directly from utils instead of `utils.clock`, `utils.retries`, `utils.profiler` etc.

Again, this is completely optional and only benefits the people using the library (which is currently just Snuba) but thought I would mention it.